### DISCUSSION TITLE
Adding Hover Effect on Social Icons if Footer in website

### DISCUSSION THREAD

**Discussion by junee1206** [chakra-ui/chakra-ui#8705] [General]
### Description

Added a `hoverColor` property to each link in the links array and updated the `FooterLink` component to change the icon color on hover using Chakra UI’s _hover pseudo-prop.

### Problem Statement/Justification

Added a hover effect to the icons in the footer to enhance user experience and provide visual feedback when interacting with the links.

### Proposed Solution or API

https://github.com/chakra-ui/chakra-ui/pull/8703

### Additional Information

https://www.awesomescreenshot.com/video/30157282?key=1bc110c37d53f0d195a93f7fe8194308

**Comment by segunadebayo** [comment]
Closing. This is stale and we are not acting on it.
